src/connectedvmware/azext_connectedvmware/_params.py

```diff
@@ -346,12 +352,77 @@ def load_arguments(self, _):
             help="Username to use for connecting to the vm.",
         )
         c.argument(
-            'password',
-            options_list=['--password'],
+            'password', options_list=['--password'],
+            help="Username password credentials to use for connecting to the VM.",
+        )
+
+    with self.argument_context('connectedvmware vm guest-agent show') as c:
+        c.argument(
+            'password', options_list=['--password'],
             help="Username password credentials to use for connecting to the VM.",
         )
 
     with self.argument_context('connectedvmware vm guest-agent show') as c:
         c.argument(
             'vm_name', options_list=['--vm-name'], help="Name of the VM.",
         )
+
+    with self.argument_context('connectedvmware vm extension list') as c:
+        c.argument('vm_name', options_list=['--vm-name'], type=str, help='The name of the vm containing the extension.')
+        c.argument('expand', type=str, help='The expand expression to apply on the operation.')
```

Comment:
It is recommended to use the options of expand expression as different parameters instead of using `--expand` directly, which can avoid users entering complex expressions.
Here is a similar feature, maybe you can refer to it https://github.com/Azure/azure-cli/issues/20155

Reply:
Fix